oauth2/src/main/java/com/okta/spring/boot/oauth/env/OIDCMetadata.java

```diff
@@ -0,0 +1,76 @@
+package com.okta.spring.boot.oauth.env;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.ResponseEntity;
+
+public class OIDCMetadata {
+
+    private final String clientAuthenticationMethod = "none";
+    private final String scope = "profile,email,openid";
```

Comment:
What is the rationale behind hardcoding the scopes and these values in specific? The SDK user should ideally be able to request any additional scopes including custom scopes?

Reply:
these were already hardcoded in `OktaOAuth2PropertiesMappingEnvironmentPostProcessor` -> `otkaForcePkcePropertySource` and `defaultOktaScopesSource`, I just moved them here. They seem like default values